foundation-extensions/lifecycle-for-edge-network/README.md

```diff
@@ -0,0 +1,93 @@
+# Lifecycle for Edge Network
+
+The Adobe Experience Platform Lifecycle mobile extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/).
+
+
+## Add AEP Mobile Lifecycle Details field group to your XDM Schema in Adobe Experience Platform
+
+Include the AEP Mobile Lifecycle Details field group to the schema defined in your Edge configuration data stream. The AEP Mobile Lifecycle field group defines all the metrics auto-collected by the Lifecycle for Edge Network extension. 
+
+1. In Adobe Experience Platform, log into your organization.
+2. Under **Schemas** section, select the **Browse** tab and search for the schema used in the application's Edge configuration.
+3. In the **Field groups** section, select **Add**
+4. Search for **AEP Mobile Lifecycle Details**, select it then click **Add field groups**
+5. Click **Save**
+
+![](../../.gitbook/assets/xdm-schema-add-mobile-lifecycle-details.png)
+
+
+## Configure the Adobe Experience Platform Lifecycle extension in Experience Platform Launch
+
+{% hint style="info" %}
+The Lifecycle for Edge Network extension is part of the Mobile Core extension. There is no separate card for the Lifecycle for Edge Network extension in the Launch Extensions Catalog.
+{% endhint %}
+
+{% hint style="warning" %}
+The configuration setting "Session Timeout" in the Mobile Core extension configuration is used when collecting Lifecycle session data for Analytics and is not used for the Lifecycle for Edge Network extension.
+{% endhint %}
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Mobile Core** extension, and click **Install**.
+3. There are no configuration settings for **Lifecycle for Edge Network**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+In addition, use the following steps to [Configure the AEP Edge Network extension in Launch](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension#configure-the-edge-network-extension-in-launch).
+
+![AEP Mobile Core extension configuration](../../.gitbook/assets/mobile-core-launch-configuration.png)
+
+## Configure Forward to Edge Rule in Launch
```

Comment:
Can we change this to Configure a Rule in Launch to forward Lifecycle metrics to Platform